.github/ISSUE_TEMPLATE/mystery_event.yml

```
@@ -81,27 +84,37 @@ body:
         - LABORATORY
         - END
         - OTHER (please specify)
-
     validations:
       required: true
   - type: textarea
     id: options
     attributes:
       label: Options offered to the player
-      placeholder: Remember that only four options can be displayed at a time
+      description: A maximum of four options can be displayed at a time
+      placeholder: Remember that only a maximum of up to four options can be displayed at a time
```

```suggestion
      placeholder: Remember that only up to four options can be displayed at a time
```

Thinking: "a maximum of" already implies "up to X"